docs/configuration/index.md

```diff
@@ -2054,7 +2054,7 @@ Supported query contexts:
 |`druid.router.tierToBrokerMap`|Queries for a certain tier of data are routed to their appropriate Broker. This value should be an ordered JSON map of tiers to Broker names. The priority of Brokers is based on the ordering.|{"_default_tier": "<defaultBrokerServiceName>"}|
 |`druid.router.defaultRule`|The default rule for all datasources.|"_default"|
 |`druid.router.pollPeriod`|How often to poll for new rules.|PT1M|
-|`druid.router.sql.enable`|Enable routing of SQL queries. Possible values are `true` and `false`. When set to `true`, the Router uses the provided strategies to determine the broker service for a given SQL query.|`false`|
+|`druid.router.sql.enable`|Enable routing of SQL queries using strategies. Possible values are `true` and `false`. When set to `true`, the Router uses the provided strategies to determine the broker service for a given SQL query. When set to `false`, the Router uses the `defaultBrokerServiceName`.|`false`|
```

Comment:
can you also add what strategies are supported?

Reply:
The details of strategies are present in docs/design/router.md. I figured we only need a summary here.